src/chrome/content/rules/Reethus-adeline.de.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="Reethus-adeline.de">
+	<target host="www.reethus-adeline.de" />
+	<target host="reethus-adeline.de" />
+
+	<securecookie host="^(www\.)?reethus-adeline\.de$" name=".*" />
+
+	<rule from="^http://(www\.)?reethus-adeline\.de/" to="https://$1reethus-adeline.de/" />
```

Comment:
Replace this line by

		<rule from="^http://(?:www\.)?reethus-adeline\.de/" to="https://www.reethus-adeline.de/" />

because https://reethus-adeline.de/ contains the following base line in the `<HEAD>` section:

	<base href="https://www.reethus-adeline.de/" />

This avoids a hostname change when navigating to any other page of that site.